TheMovieDatabase/Services/AuthorizationService.swift

```diff
@@ -0,0 +1,287 @@
+//
+//  AuthorizationService.swift
+//  TheMovieDatabase
+//
+//  Created by Evgeny Syrtsov on 4/17/20.
+//  Copyright © 2020 Evgeny Syrtsov. All rights reserved.
+//
+
+import Foundation
+import Locksmith
+
+final class AuthorizationService {
+
+    // MARK: - Properties
+
+    static var sessionId: String? {
+        return Locksmith.sessionId
+    }
+    private let moviesLoadingService = MoviesLoadingService()
+    private let profileService = ProfileService()
+
+    // MARK: - Methods
+
+    // swiftlint:disable cyclomatic_complexity
```

Comment:
Oh you're too tricky
You can't just disable swiftlint everywhere where you have warning and can't fix it

Reply:
its temporary solution